.circleci/config.yml

```diff
@@ -33,7 +33,8 @@ defaults: &defaults
          . ci/bin/activate
          pip install jsmin
          pip install six
-         pip install bokeh
+         pip install "numpy<2"
+         pip install "bokeh<=2.4"
```

Comment:
Can we raise an issue to remove this pin requirement?

Reply:
Yep, as per the description I will do that once it merges :)